x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import { reject, isUndefined } from 'lodash';
+import { SearchResponse, Client } from 'elasticsearch';
```

Comment:
Now I feel slightly embarrassed that I didn't do the basic typing when I wrote most of this code.  Thanks for upgrading all this stuff!!!

Reply:
hah nah, don't be silly, we did it to ourselves by not having this rule in place in the first place :)